### ISSUE TITLE
[Anon] Bug report: Chaos - Beastmen.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/warhammer-age-of-sigmar#875]
**File:** Chaos - Beastmen.catz

**BattleScribe version:** 2.02.01

**Platform:** Android

**Dropbox:** No

**Description:** Desolating Beastherd does not let you pick gors or chariots, which are required by the battalion.
Cost of doombull  and Shagoth are incorrect.
Unable to pick God's allegiance when selecting God specific battalions.

**Comment by cartag** [comment]
Doombull is correct, Shoggoth got corrected, Desolating Beastherd corrected. 

Being unable to pick god allegiance is a known problem, waiting on BS fixes to correctly implement. Duplicate of #862
